constants/sprite_constants.asm

```diff
@@ -195,27 +195,28 @@
 	const SPRITE_ICE_BOULDER ; bf
 DEF NUM_OVERWORLD_SPRITES EQU const_value - 1
 
-	; c0-ef = 48 unused
+	; c0-ee = 47 unused
 
 ; special GetMonSprite values (see engine/overworld/overworld.asm)
-	const_next $f0
+	const_next $ef
 DEF SPRITE_POKEMON EQU const_value
-	const SPRITE_MON_ICON ; f0
-	const SPRITE_MON_DOLL_1 ; f1
-	const SPRITE_MON_DOLL_2 ; f2
-	const SPRITE_DAYCARE_MON_1 ; f3
-	const SPRITE_DAYCARE_MON_2 ; f4
-	const SPRITE_GROTTO_MON ; f5
+	const SPRITE_MON_ICON ; ef
+	const SPRITE_MON_DOLL_1 ; f0
+	const SPRITE_MON_DOLL_2 ; f1
+	const SPRITE_DAYCARE_MON_1 ; f2
+	const SPRITE_DAYCARE_MON_2 ; f3
+	const SPRITE_GROTTO_MON ; f4
 
 ; wVariableSprites indexes (see ram/wramx.asm)
 DEF SPRITE_VARS EQU const_value
-	const SPRITE_CONSOLE ; f6
-	const SPRITE_DOLL_1 ; f7
-	const SPRITE_DOLL_2 ; f8
-	const SPRITE_BIG_DOLL ; f9
-	const SPRITE_FUCHSIA_GYM_1 ; fa
-	const SPRITE_FUCHSIA_GYM_2 ; fb
-	const SPRITE_FUCHSIA_GYM_3 ; fc
-	const SPRITE_FUCHSIA_GYM_4 ; fd
-	const SPRITE_COPYCAT ; fe
-	const SPRITE_JANINE_IMPERSONATOR ; ff
+	const SPRITE_CONSOLE ; f5
+	const SPRITE_DOLL_1 ; f6
+	const SPRITE_DOLL_2 ; f7
+	const SPRITE_BIG_DOLL ; f8
+	const SPRITE_FUCHSIA_GYM_1 ; f9
+	const SPRITE_FUCHSIA_GYM_2 ; fa
+	const SPRITE_FUCHSIA_GYM_3 ; fb
+	const SPRITE_FUCHSIA_GYM_4 ; fc
+	const SPRITE_COPYCAT ; fd
+	const SPRITE_JANINE_IMPERSONATOR ; fe
+	const SPRITE_LINK_TRAINER ; ff
```

Comment:
I'll double-check later if `SPRITE_LINK_TRAINER` should be first or last, in order not to shift the other variable sprites in WRAM.